src/index.js

```diff
@@ -1,6 +1,10 @@
 const { ApolloLink } = require('apollo-link');
 const formatMessage = require('./formatMessage');
 
+
+const logGroupCollapsed = console.groupCollapsed || () => {};
```

Comment:
How about `console.groupCollapsed || console.log`?
The group info contains useful information that might be better to print out.